static/src/javascripts/projects/membership/payment-form.js

```diff
@@ -61,14 +60,14 @@ define([
      */
     StripePaymentForm.prototype.updateCard = function (card) {
         var cardTypeClassName = card.type.toLowerCase().replace(' ', '-');
-        var cardTypeElem = this.config.DOM.CARD_TYPE;
-        $(this.config.DOM.CARD_LAST4).text(card.last4);
+        var cardTypeElem = this.DOM.CARD_TYPE;
```

Comment:
I would argue that to prevent this kind of thing happening again, you should just move the `config` object off the prototype. It can hang directly off of `StripePaymentForm` (`StripePaymentForm.config`) like a "static" property. As discussed, methods on the prototype are fine, but data is generally a bad idea